assertions/src/main/java/io/camunda/zeebe/process/test/assertions/DeploymentAssert.java

```diff
@@ -118,4 +119,48 @@ public ProcessAssert extractingProcessByResourceName(final String resourceName)
 
     return new ProcessAssert(matchingProcesses.get(0), recordStream);
   }
+
+  /**
+   * Extracts the form with the given form ID
+   *
+   * @param formId form ID to look up
+   * @return this {@link JobAssert}
```

Comment:
❌ JavaDoc return type of this and the method below are wrong. Should be `FormAssert`.